### PR TITLE
Allow accepting image for the members in OpenStack builder

### PR DESCRIPTION
**Pull request by jorgemarey**
Hi, in openstack you can share an image for some members setting them in the **image_members** parameter, but then, for the members to see it on their proyect, you need to share the image ID with them, and they need to perform an accept action. This is documented [here](https://docs.openstack.org/image-guide/share-images.html)

This PR adds a configuration parameter **image_auto_accept_members** that performs this action after sharing the image with those members. This needs a user with special privileges but if you use it on your image pipeline, this could simplify it.

### CODE REVIEW

File: builder/openstack/image_config.go [hunk 1]
```diff
@@ -22,6 +22,9 @@ type ImageConfig struct {
 	// usually a project (also called the "tenant") with whom the image is
 	// shared.
 	ImageMembers []string `mapstructure:"image_members" required:"false"`
+	// True to perform the image accept so the member can see the image in his
```
Comment (SwampDragons) [inline builder/openstack/image_config.go]: I'd like to see a bit more detail here, including the fact that for it to work your build user needs to have special privileges. Also, make it clear here that by default the value is "false"

Comment (jorgemarey) [reply builder/openstack/image_config.go]: Makes sense.

**Comment by jorgemarey** [comment]
I just updated the documentation with those changes. If you think there're other things that needs to change, let me know. Thanks!

File: builder/openstack/image_config.go [hunk 2]
```diff
@@ -22,6 +22,10 @@ type ImageConfig struct {
...
```
Comment (SwampDragons) [inline builder/openstack/image_config.go]:
```suggestion
	// When true, perform the image accept so the member can see the image in their
```

File: builder/openstack/image_config.go [hunk 3]
```diff
@@ -22,6 +22,10 @@ type ImageConfig struct {
...
+	// project. This requires an user with priveleges both in the build project and
```
Comment (SwampDragons) [inline builder/openstack/image_config.go]:
```suggestion
	// project. This requires a user with priveleges both in the build project and
```

Comment (jorgemarey) [reply builder/openstack/image_config.go]: Fixed!

File: builder/openstack/image_config.go (hunk 2 above)
Comment (jorgemarey) [reply builder/openstack/image_config.go]: Fixed!

**Pull request merged**